docs/index.asciidoc

```diff
@@ -9,14 +9,10 @@ include::installation.asciidoc[]
 
 include::connecting.asciidoc[]
 
+include::usage.asciidoc[]
+
 include::configuration.asciidoc[]
 
 include::operations.asciidoc[]
 
-include::build/classes.asciidoc[]
-
-include::helpers.asciidoc[]
-
 include::release-notes.asciidoc[]
-
-include::redirects.asciidoc[]
```

Comment:
@ezimuel This line also needs to be added back to pass the docs-ci. Sorry for not catching it in the first round.

Reply:
Done